### ISSUE TITLE
An error in executing `grader.check_all()`

### ISSUE THREAD

**Issue by spring-haru** [ucbds-infra/otter-grader#746]
**Describe the bug**
In JupyterLite, executing `grader.check_all()` in a student .ipynb generates the following error:

```
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
Cell In[5], line 1
----> 1 grader.check_all()

File /lib/python3.11/site-packages/otter/check/utils.py:184, in grading_mode_disabled(wrapped, self, args, kwargs)
    182 if type(self)._grading_mode:
    183     return
--> 184 return wrapped(*args, **kwargs)

File /lib/python3.11/site-packages/otter/check/utils.py:217, in logs_event.<locals>.event_logger(wrapped, self, args, kwargs)
    215 except Exception as e:
    216     self._log_event(event_type, success=False, error=e)
--> 217     raise e
    219 if ret is None:
    220     ret = LoggedEventReturnValue(None)

File /lib/python3.11/site-packages/otter/check/utils.py:213, in logs_event.<locals>.event_logger(wrapped, self, args, kwargs)
    208 """
    209 Runs a method, catching any errors and logging the call. Returns the unwrapped return value
    210 of the wrapped function.
    211 """
    212 try:
--> 213     ret: Optional[LoggedEventReturnValue[T]] = wrapped(*args, **kwargs)
    215 except Exception as e:
    216     self._log_event(event_type, success=False, error=e)

File /lib/python3.11/site-packages/otter/check/notebook.py:537, in Notebook.check_all(self)
    531 """
    532 Runs all tests on this notebook. Tests are run against the current global environment, so
    533 any tests with variable name collisions will fail.
    534 """
    535 self._log_event(EventType.BEGIN_CHECK_ALL)
--> 537 tests = list_available_tests(self._tests_dir, self._notebook)
    539 global_env = inspect.currentframe().f_back.f_back.f_back.f_globals
    541 self._logger.debug(f"Found available tests: {', '.join(tests)}")

File /lib/python3.11/site-packages/otter/check/utils.py:274, in list_available_tests(tests_dir, nbmeta_config)
    271     tests = map(get_stem, list_test_files(tests_dir))
    273 else:
--> 274     tests = list(nbmeta_config.tests.keys())
    276 return sorted(tests)

AttributeError: 'str' object has no attribute 'tests'
```

**To Reproduce**
Execute the attached .ipynb file in JupyterLite or

https://haruyama-kobeu.github.io/jlite/

which I used

**Expected behavior**
No such error

**Versions**
otter-grader  5.2.1

**Additional context**
Here is the file, but please remove .txt

[test.ipynb.txt](https://github.com/ucbds-infra/otter-grader/files/12773466/test.ipynb.txt)

**Comment by chrispyles** [comment]
@spring-haru I've pushed a fix for this bug to my GH repo and in the zipped wheel file below. Can you unzip and install the wheel file in your Jupyterlite distro and let me know if you run into any more issues so they can also be included in v5.2.2?

[otter_grader-5.2.1-py3-none-any.whl.zip](https://github.com/ucbds-infra/otter-grader/files/12775819/otter_grader-5.2.1-py3-none-any.whl.zip)

To install it, you can drag and drop the unzipped wheel file into the Jupyterlite file pane in your browser and then run

```python
import piplite
await piplite.install("emfs:otter_grader-5.2.1-py3-none-any.whl")
```

**Comment by spring-haru** [comment]
It worked without an error🎉. Thank you㊗️
